test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -107,7 +107,7 @@ def test_successful_purchase_using_ach
   end
 
   def test_successful_purchase_with_options
-    options = @options.merge(metadata: @metadata, ewallet_id: 'ewallet_1a867a32b47158b30a8c17d42f12f3f1')
+    options = @options.merge(metadata: @metadata, ewallet_id: 'ewallet_897aca846f002686e14677541f78a0f4')
```

Comment:
After updating the field name to `ewallet`, this test failed because evidently this ID doesn't exist on the test account, so I created a new one